firmware/applications/SignBuddy/flex.c

```diff
@@ -81,9 +83,13 @@ void flex_start_read(void)
   xTaskNotifyGive(s.task_handle);
 }
 
-void flex_get_value(uint16_t *measurement, uint8_t index)
+void flex_data_get(Sample_FlexData *data)
 {
-  *measurement = s.flex_values[index];
+  data->flex_thumb = s.flex_values[0];
```

Comment:
do we not have defines for which index in the array corresponds to which finger?